Makefile

```diff
@@ -503,3 +503,87 @@ install-wwhrd:
 
 check-licenses: install-wwhrd
 	wwhrd check -f .wwhrd.yml
+
+# Time Series Benchmark Suite (TSBS) for VictoriaMetrics
+# This command runs a complete benchmark cycle:
+# 1. Builds TSBS tools
+# 2. Generates sample time series data
+# 3. Loads data into VictoriaMetrics
+# 4. Generates benchmark queries
+# 5. Runs the queries and measures performance
+#
+# REQUIREMENTS:
+# - VictoriaMetrics must be running at http://localhost:8428 before running this benchmark
+#
+# SCALE:
+# - Adjust --scale parameter in tsbs-generate-data and tsbs-generate-queries to increase/decrease load
+# - Adjust --workers parameter to control concurrency
+#
+# DOCUMENTATION:
+# See https://github.com/timescale/tsbs/blob/master/docs/victoriametrics.md for details
+tsbs: tsbs-build tsbs-generate-data tsbs-load-data tsbs-generate-queries tsbs-run-queries
+
+# The default parameters below are chosen based on the desired scale and time range.
+# With these parameters, the benchmark will generate 1 billion samples:
+#
+# - Each instance emits 10 unique metrics and 4K instances, therefore, emit 40K unique metrics
+# - Within the data file, each line contains 10 samples from one instance
+# - Metrics are emitted every 10s interval and there are 3600*24*3 / 10 = ~26K 10s intervals within 3 days
+# - Total number of lines, therefore: 4K machines × 26K intervals = ~100M
+# - And total number of samples: 40K metrics × 26K intervals = ~1B
+TSBS_SCALE := 4000
+TSBS_START := $(shell date -u -d "3 days ago 00:00:00" +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
this command won't work on macOS :(

Reply:
Implemented a better approach and added a comment to the place where the date is generated. It will work on both systems like linux and macos